buildscripts/packaging/MacOS/make_dmg.sh

```
@@ -112,15 +112,13 @@ function change_rpath() {
 #tip: increase the size if error on copy or macdeployqt
-hdiutil create -size 750m -fs HFS+ -volname ${VOLNAME} ${WORKING_DIRECTORY}/${DMGNAME}
+hdiutil create -size 800m -fs APFS -volname ${VOLNAME} ${WORKING_DIRECTORY}/${DMGNAME}
 
 # Mount the disk image
-hdiutil attach ${WORKING_DIRECTORY}/${DMGNAME}
-
-# Obtain device information
-DEVS=$(hdiutil attach ${WORKING_DIRECTORY}/${DMGNAME} | cut -f 1)
-DEV=$(echo $DEVS | cut -f 1 -d ' ')
-VOLUME=$(mount |grep ${DEV} | cut -f 3 -d ' ')
+VOLUME="/Volumes/${VOL_NAME}"
```

Shellcheck (0.11.0)

[info] Possible misspelling: VOL_NAME may not be assigned. Did you mean VOLNAME?

(SC2153)